File: Wilna/Trinket/3line.js
const fs = require('node:fs');
var LineX, LineY, LineZ

fs.readFile('results.txt', 'utf8', (err, data) => {
  data = data.split("\r\n")
  for (let i = 0; i < data.length; i++) {
    LineZ = LineY
    LineY = LineX
    LineX = data[i]
  }
  console.log(LineZ)
  console.log(LineY)
  console.log(LineX)
});

File: Wilna/Trinket/3line.py
# def LastLines(file):
#     LineX, LineY, LineZ = "", "", ""
#     for j in range(len(f := (open(file, "r")).readlines())):
#         LineZ, LineY, LineX = LineY, LineX, f[j].removesuffix("\n")
#     return(f"{LineZ}\n{LineY}\n{LineX}")
# print(LastLines("results.txt"))

# def LastLines(file):
#     for j in range(len(f := (open(file, "r")).readlines())): f[j] = f[j].removesuffix("\n")
#     return str(f[len(f)-3:len(f)]).replace("[", "").replace("'", "").replace(",", "").replace("]", "").replace(" ", "\n")
# print(LastLines("results.txt"))

# for j in range(len(f := (open("results.txt", "r")).readlines())): f[j] = f[j].removesuffix("\n")
# print(str(f[len(f)-3:len(f)]).replace("[", "").replace("'", "").replace(",", "").replace("]", "").replace(" ", "\n"))

f = (open("results.txt", "r")).readlines()
print((LineX := f[len(f)-3:len(f)][0]) + (LineY := f[len(f)-3:len(f)][1]) + (LineZ := f[len(f)-3:len(f)][2]))